Issue 18 home page

Sivanwol opened the pull request.

Sivanwol committed:
fix: remove bootsrap

Sivanwol committed:
feat: work on the general layout of the page

Sivanwol committed:
feat: fix formatting

Sivanwol committed:
feat: formatting

Sivanwol committed:
fix: issue with the flag of stand alone been removed

Sivanwol committed:
feat: fix issue with 404 page and update and add effect

Sivanwol committed:
fix: deployment

Sivanwol committed:
fix: cd issue... more to come

Sivanwol committed:
fix: cd fix (what it didnt do much)

swarnava commented:
#### 📝 Changed routes:


| [**/\[locale\]**](https://nextjs-template-9hman2u50.preview.vercel-support.app/[locale]) | [**/**](https://nextjs-template-9hman2u50.preview.vercel-support.app/) |
|:-:|:-:|
|<a href="https://nextjs-template-9hman2u50.preview.vercel-support.app/[locale]"><img src="https://images.weserv.nl?url=https://sjc.microlink.io/C9VR2VFvy2ahXMkyWkd30QX9493cB8OkS4FmeDUgbRmd6shQ0dLAYiqCnwkXPXcQ4VUQKUJMgOiO1ItWxn57cQ.png&w=600" alt="Screenshot of /[locale]" width="300"></a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/MW65irlBbAHCtRwwPZ98BCEHo-bpmkCJt5WRhPalp8kZfkP_q6laowy30dInM6u-suZyqJr29lGcKecKaqwBrg.png">(view full size)</a> |<a href="https://nextjs-template-9hman2u50.preview.vercel-support.app/"><img src="https://images.weserv.nl?url=https://sjc.microlink.io/E3ZQqSlWDFDiVy7sq-UGNKlc5CC50WhjzEs7OvlhPNvz1rreDEjoCfFjm5wl-2_nCw9nQh_V9bMG67Km51L_hw.png&w=600" alt="Screenshot of /" width="300"></a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/AqvbJizfT2qla7lQ6JAXC04sfGc2CNLWebZKJYmTvFDC1HtdPGSmzxBNPhhTWteJQdP8CkOD8i6qi2ksLSmClA.png">(view full size)</a> |







Commit 2f634e2e6dbd554db14f487911482509701fee79 (https://nextjs-template-9hman2u50.preview.vercel-support.app).

Sivanwol committed:
fix: try once more the update of the changes

Sivanwol committed:
fix: possible issue so i remove Hypertune

Sivanwol committed:
fix: reformatting

Sivanwol committed:
fix: issue with auto formatting

Sivanwol committed:
fix: once more yes faield

Sivanwol committed:
fix: small fix and more test see how i can fix cd issue

Sivanwol committed:
fix: remove thgis libv

Sivanwol committed:
fix: cd issue maybe will be fixed

Sivanwol committed:
fix: testing

Sivanwol committed:
fix: more work on this issue

Sivanwol committed:
chore: remove this it work

Sivanwol committed:
chore: more testing

Sivanwol committed:
fix: got it running adding the footer

Sivanwol committed:
fix: add back the effects

Sivanwol committed:
fix: add fadein

Sivanwol committed:
chore: update the headers

Sivanwol committed:
fix: check if this the issue

Sivanwol committed:
fix: remove something see if fix this issue

Sivanwol committed:
fix: more work and fix

Sivanwol committed:
fix: this crap

Sivanwol committed:
fix: update logic

Sivanwol committed:
fix: update logic

Sivanwol committed:
chore: once more fix  for this crap

Sivanwol committed:
fix: issue with cd need debug disable breand

Sivanwol committed:
chore: update logic

Sivanwol committed:
fix: woops

Sivanwol committed:
fix: try fix issue with logo size

Sivanwol committed:
fix: few bugs that i located

Sivanwol committed:
feat: more work on the system

Sivanwol committed:
chore: update layout

Sivanwol committed:
fix: try to fix logo

dependabot[bot] committed:
chore(deps): bump @tanstack/react-virtual from 3.8.1 to 3.8.3

Bumps [@tanstack/react-virtual](https://github.com/TanStack/virtual/tree/HEAD/packages/react-virtual) from 3.8.1 to 3.8.3.
- [Release notes](https://github.com/TanStack/virtual/releases)
- [Commits](https://github.com/TanStack/virtual/commits/v3.8.3/packages/react-virtual)

---
updated-dependencies:
- dependency-name: "@tanstack/react-virtual"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Sivanwol committed:
chore: more work on the base of the front page

Sivanwol committed:
chore: add the skills images

Sivanwol committed:
chore: fix conflict

Sivanwol committed:
fix: lock issue conflict

Sivanwol committed:
chore: remove unused and broken stuff

Sivanwol committed:
chore: more work on the layout of the pages

Sivanwol committed:
chore: update and add speed dialer

Sivanwol committed:
chore: fix issue with lagecy crap

Sivanwol committed:
chore: start working on the contact us form

Sivanwol committed:
chore: update code segment

Sivanwol committed:
chore: update form

Sivanwol committed:
chore: update form

Sivanwol committed:
chore: more fix'es

Sivanwol committed:
fix: break cd

Sivanwol committed:
chore: add basic board of the mini game within the homepage

Sivanwol committed:
chore: more work try fix issue with HTMLVIDEOElement is undefined

Sivanwol committed:
revert: not only didnt work it was stupid

Sivanwol committed:
chore: now it works

Sivanwol committed:
chore: update the layout

Sivanwol committed:
chore: update what been missed

Sivanwol committed:
chore: update issue with remove of the canvas

Sivanwol committed:
chore: update Text

Sivanwol committed:
fix: small fix for vercel

Sivanwol committed:
chore: update the system perf

Sivanwol committed:
fix: responsive issue

Sivanwol committed:
fix: make contact form meaning full

Sivanwol committed:
ci: ignore error for this part

Sivanwol committed:
ci: delete this crap

Sivanwol committed:
fix: issue with responsive

Sivanwol committed:
chore: remove unused environment variables and deprecated code

- Remove unused environment variables related to Hypertune integration
- Delete deprecated getHypertune function
- Remove HypertuneProvider from providers.tsx

Sivanwol committed:
chore: remove unused environment variables and deprecated code

Sivanwol committed:
chore: remove unused "hypertune" package dependency

Sivanwol committed:
chore: remove unused "lru-cache" package dependency

Sivanwol committed:
chore: test
